src/WebView.vala

```diff
@@ -160,6 +160,21 @@ public class Mail.WebView : WebKit.WebView {
         }
     }
 
+    public new void load_plain_text (string plain_text) {
+        // Instead of calling base.load_plain_text, use Camel's ToHTML filter to
+        // convert text to HTML. This gives us some niceties like clickable URLs
+        // and email addresses.
+
+        // Explanation of ToHTML flags:
+        // https://wiki.gnome.org/Apps/Evolution/Camel.MimeFilter#Camel.MimeFilterToHtml
```

Comment:
Please use multiline comment style:
```
/*
 * Instead of calling base.load_plain_text, use Camel's ToHTML filter to
 * convert text to HTML. This gives us some niceties like clickable URLs
 * and email addresses.
 *
 * Explanation of ToHTML flags:
 * https://wiki.gnome.org/Apps/Evolution/Camel.MimeFilter#Camel.MimeFilterToHtml
 */```